<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastre_parcelle_form</class>
 <widget class="QDialog" name="cadastre_parcelle_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastre - Informations Parcelle</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>620</width>
        <height>561</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Résumé</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QTextEdit" name="parcelleInfo">
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Propriétaires</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTextEdit" name="proprietairesInfo">
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Subdivisions</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTextEdit" name="subdivisionsInfo">
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Locaux</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTextEdit" name="locauxInfo">
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Actions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btCentrer">
              <property name="text">
               <string>Centrer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btZoomer">
              <property name="text">
               <string>Zoomer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btSelectionner">
              <property name="text">
               <string>Sélect.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="btParcellesProprietaire">
              <property name="text">
               <string>Sélectionner les parcelles du propriétaire</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Export</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="combo_for_third_party">
            <property name="toolTip">
             <string>Lorsque cette case est cochée, les dates et lieux de naissance des propriétaires ne sont pas exportés dans les relevés PDF.</string>
            </property>
            <property name="text">
             <string>Les relevés sont destinés à des tiers</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btExportParcelle">
              <property name="text">
               <string>Relevé parcellaire</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="btExportProprietaire">
              <property name="text">
               <string>Relevé de propriété</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbExportAllCities">
              <property name="toolTip">
               <string>Cocher cette case permet d'exporter le relevé de propriété pour toutes les communes. Cela peut être très long !</string>
              </property>
              <property name="text">
               <string>Exporter pour toutes les communes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>btExportParcelle</tabstop>
  <tabstop>btCentrer</tabstop>
  <tabstop>btZoomer</tabstop>
  <tabstop>btSelectionner</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
